packages/flutter/lib/src/material/input_chip.dart

```diff
@@ -113,13 +113,11 @@ class InputChip extends StatelessWidget
     this.showCheckmark,
     this.checkmarkColor,
     this.avatarBorder = const CircleBorder(),
-    @Deprecated(
-      'Migrate to deleteButtonTooltipMessage. '
-      'This feature was deprecated after v2.10.0-0.3.pre.'
-    )
+    @Deprecated('Migrate to deleteButtonTooltipMessage. '
```

Comment:
same in this file

Reply:
Thank you for giving me your precious time. Now I fixed all